Manifests/ManagedPreferencesApplications/com.almenscorner.supportcompanion.plist

```diff
@@ -530,6 +542,163 @@
 			<key>pfm_type</key>
 			<string>array</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Configures the desktop info widget.</string>
+			<key>pfm_name</key>
+			<string>ShowDesktopInfo</string>
+			<key>pfm_title</key>
+			<string>Show Desktop Info</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<string>TopRight</string>
+			<key>pfm_description</key>
+			<string>Configures the position of the desktop info. Defaults to TopRight.</string>
+			<key>pfm_name</key>
+			<string>DesktopPosition</string>
+			<key>pfm_range_list</key>
+			<array>
+				<string>TopLeft</string>
+				<string>TopRight</string>
+				<string>BottomLeft</string>
+				<string>BottomRight</string>
+			</array>
+			<key>pfm_range_list_titles</key>
+			<array>
+				<string>Top Left</string>
+				<string>Top Right</string>
+				<string>Bottom Left</string>
+				<string>Bottom Right</string>
+			</array>
+			<key>pfm_title</key>
+			<string>Desktop Position</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Configures the level of information to show on the desktop. Defaults to Full.</string>
+			<key>pfm_name</key>
+			<string>DesktopInfoLevel</string>
+			<key>pfm_range_list</key>
+			<array>
+				<string>Full</string>
+				<string>Minimal</string>
+				<string>Hardware</string>
+				<string>Custom</string>
+			</array>
+			<key>pfm_range_list_titles</key>
```

Comment:
`pfm_range_list_titles` is unnecessary here if the raw values are identical.

Reply:
I have removed the list titles